### ISSUE TITLE
Bugs in latest versions of p5.sound.js

### ISSUE THREAD

**Issue by karneaud** [processing/p5.js#1281]
Been trying to understand what is going on. I ran this code a couple times and was able to execute successfully then today I get a bunch of errors that I have never seen before

In running the p5 framework in Latest Chrome and Safari I get the following error

> Uncaught InvalidAccessError: Failed to execute 'disconnect' on 'AudioNode': the given destination is not connected.Master @ p5.sound.js:176(anonymous function) @ p5.sound.js:196(anonymous function) @ p5.sound.js:218

I'm not sure why this is when running a simple code 

```
var cv, img, s = function(p){
  p.preload = function()
  {
    img = p.loadImage('../img/heineken.png', function(img){
      img = img;
    },function(){
    })
  }
  p.setup = function() {
    cv = p.createCanvas(p.windowWidth, p.windowHeight);
  }

  p.draw = function () {
    p.background(100);
  }
}, pt5 = new p5(s);

```

What is the most stable release that can be used here?

**Comment by therewasaguy** [comment]
Which version of p5.sound are you using? The latest version is v0.3.0. A bug like this was [fixed in 0.1.9](https://github.com/processing/p5.js-sound/issues/45). It's possible there is a new bug but I'm not able to reproduce it with the code you posted.

**Comment by karneaud** [comment]
Please disregard this post. I've been having problems installing and compiling p5js using bower.
